helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -170,6 +176,13 @@
               "const": true
             }
           }
+        },
+        {
+          "properties": {
+            "tracesEnabled": {
```

Comment:
What's this?

Reply:
@sbylica-splunk this allows you a setup for splunkPlatform with only traces enabled.
Without it you must enable either logs or traces. Do we want to remove this change?

Reply:
Ah, makes sense now that I'm looking at it. Is there any particular reason for this change?

Reply:
@sbylica-splunk it's inconsistent when we accept either only logs or only metrics, but not traces, but we don't need it necessarily but, this can be removed